### PR TITLE
Bumped versions in pre-commit and npm configurations.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

**Review by mgaligniana — APPROVED** [review]
Puppeter has a breaking change in [20.0.0](https://github.com/puppeteer/puppeteer/blob/main/packages/puppeteer/CHANGELOG.md#2000-2023-05-02) version (Chrome for Testing instead of Chromium). Does it affect?

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/psf/black
```
Comment (smithdc1) [inline .pre-commit-config.yaml:2]: https://black.readthedocs.io/en/latest/change_log.html#integrations

Worth switching to the mirror for a 2x speed up?

`https://github.com/psf/black-pre-commit-mirror`

Comment (felixxm) [reply .pre-commit-config.yaml]: Good catch :+1: Changed.

**Comment by felixxm** [comment]
> Puppeter has a breaking change in [20.0.0](https://github.com/puppeteer/puppeteer/blob/main/packages/puppeteer/CHANGELOG.md#2000-2023-05-02) version (Chrome for Testing instead of Chromium). Does it affect?

I don't think so :thinking:

**Review by sarahboyce — APPROVED** [review]

**Comment by felixxm** [comment]
Thanks y'all for reviews :+1:

**Pull request merged**